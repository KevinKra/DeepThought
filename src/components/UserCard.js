import React from "react";
import "../styles/_UserCard.scss";
class UserCard extends React.Component {
  render() {
    return (
      <article className="userCard">
        <p>content</p>
      </article>
    );
  }
}

export default UserCard;
