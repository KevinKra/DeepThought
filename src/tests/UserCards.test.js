import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import UserCard from "../components/UserCard";

describe("<UserCard />", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<UserCard />);
  });
  it("should match the snapshot", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
