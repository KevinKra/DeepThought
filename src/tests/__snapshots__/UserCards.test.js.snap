// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UserCard /> should match the snapshot 1`] = `
<article
  className="userCard"
>
  <UserStats />
</article>
`;
